README.md

```diff
@@ -134,8 +134,8 @@ This section explains the basic usage of Windows Remote Arduino. This is an exce
         using namespace Microsoft.Maker.Serial;
 
 		//create a bluetooth connection and pass it to the RemoteDevice
-		IStream bt = new BluetoothSerial();
-        RemoteDevice arduino = new RemoteDevice( bt );
+		IStream connection = new BluetoothSerial( "MyBluetoothDevice" );
```

Comment:
You're right about using the previous syntax, we did remove the default constructor and this needs to be removed. The "MyBluetoothDevice" represents a device name or id, as either are accepted as the string constructor